src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -77,6 +87,24 @@
  *   &lt;property name=&quot;validateComments&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * void fun1() {}    // violation, 4 whitespaces before the comment starts
```

Comment:
`4 whitespaces before the comment starts`
I think 2 whitespaces is enough

Reply:
I changed the amount of whitespaces to show how it will still break for more number of whitespaces. So should I still change it?

Reply:
yes

Reply:
Ive fixed this